docs/Agent/user-guide/man_page.md

```diff
@@ -1 +1,439 @@
+- dcgm
+- pcp
+
+For a list of tool-specific options, run:
+/opt/pbench-agent/tool-scripts/<tool-name> --help
+
+[--persistent] [--transient]  
+This can be used to specify tool run type.
+
+--remotes  
+Single remote host, a list of remote hosts (comma-separated, no spaces) or an "at" sign ("@") followed by a filename. In this last case, the file should contain a list of hosts and their (optional) labels. Each line of the file should contain a hostname, optionally followed by a label separated by a comma (","); empty lines are ignored, and comments are denoted by a leading hash, or pound ("#"), character.
+
+--help  
+Show this message and exit.
+
+---
+
+### pbench-register-tool-set
+
+---
+
+**NAME**  
+pbench-register-tool-set, Register the specified toolset
+
+**SYNOPSIS**  
+`pbench-register-tool-set [--toolset=<tool-set>] [--group=<group-name>] [--interval=<interval>] [--no-install] [--remotes=<remote-host>[,<remote-host>]] [--labels=<label>[,<label>]] [<tool-set>]`
+
+`pbench-register-tool-set [--toolset=<tool-set>] [--group=<group-name>] [--interval=<interval>] [--no-install] [--remotes=@<remotes-file>] [<tool-set>]`
+
+**DESCRIPTION**  
+Register all the tools in the specified toolset.
+
+**OPTIONS**  
+ --toolset  
+Available tool sets from /opt/pbench-agent/config/pbench-agent.cfg:
+
+- heavy
+- legacy
+- light
+- medium
+
+--remotes  
+Single remote host, a list of remote hosts (comma-separated, no spaces) or an "at" sign ("@") followed by a filename. In this last case, the file should contain a list of hosts and their (optional) labels. Each line of the file should contain a hostname, optionally followed by a label separated by a comma (","); empty lines are ignored, and comments are denoted by a leading hash, or pound ("#"), character.
+
+--labels  
+Where the list of labels must match the list of remotes.
+
+--help  
+Show this message and exit.
+
+---
+
+### pbench-register-tool-trigger
+
+---
+
+**NAME**  
+pbench-register-tool-trigger, Registers the tool trigger
+
+**SYNOPSIS**  
+`pbench-register-tool-trigger[-C][-g][--start-trigger][--stop-trigger][--help]`
+
+**DESCRIPTION**  
+Registers tool with the given group and start and stop tool trigger text
+
+**OPTIONS**  
+ -C, --config PATH  
+Path to a pbench-agent configuration file (defaults to the '\_PBENCH_AGENT_CONFIG' environment variable, if defined) [required]
+
+-g, --group,--groups STR  
+Registers the trigger in the <group-name> group. If no group is specified, the 'default'
+
+--start-trigger STR  
+[required]
+
+--stop-trigger STR  
+[required]
+
+--help  
+Show this message and exit.
+
+---
+
+### pbench-results-move
+
+---
+
+**NAME**  
+pbench-results-move, Move results tarball to the server
+
+**SYNOPSIS**  
+`pbench-results-move[-C][--controller][--token][--delete][--xz-single-threaded][--show-server][--help]
+`
+
+**DESCRIPTION**  
+Move result directories to the configured Pbench server.This command is used when pushing result tarball using token.
```

Comment:
Again, "using token" is true, but that's not really the important distinction. We use an API Key authentication token when pushing results to a 1.0 Pbench Server; we use `pbench-*-results` with an `id_rsa` when pushing to a 0.69 Pbench Server.